core/src/org/pentaho/di/core/exception/IdNotFoundException.java

```diff
@@ -0,0 +1,103 @@
+/*! ******************************************************************************
```

Comment:
@dkincade what about this one: https://github.com/pentaho/pentaho-kettle/pull/1823/files#diff-043e715fee3ba6d93077bd1b66a24c41R2 - what kind of test do you mean? There is no business logic in the class to test - just getetrs and setters. Or do you maen something else? CC: @mtseu